.github/workflows/threatcrush.yml

```
@@ -0,0 +1,45 @@
+        run: |
+          set +e
+          set -uo pipefail
+          npx --yes "@profullstack/threatcrush@${THREATCRUSH_VERSION}" scan src --format sarif --output threatcrush.sarif
+          sarif_code=$?
+          if [ ! -s threatcrush.sarif ]; then
+            echo "::error::ThreatCrush produced no SARIF (exit ${sarif_code}) — this diff was NOT scanned"
+            exit 1
+          fi
+      - name: Upload SARIF
+        uses: github/codeql-action/upload-sarif@v4
+        if: always() && hashFiles("threatcrush.sarif") != ""
```

actionlint (1.7.12)

[error] got unexpected character '"' while lexing expression, expecting 'a'..'z', 'A'..'Z', '_', '0'..'9', ''', '}', '(', ')', '[', ']', '.', '!', '<', '>', '=', '&', '|', '*', ',', ' '. do you mean string literals? only single quotes are available for string delimiter

(expression)